docs/hub-deployment-guide/deploy-support/configure-support.md

```diff
@@ -12,47 +12,16 @@ then a lot of these files will have already been created for you and you do not
 need to recreate them, only update them if required.
 ```
 
-## Create a `support.values.yaml` file in your chosen cluster folder
+## Make sure the `support.values.yaml` file in your chosen cluster folder is correctly configured
```

Comment:
```suggestion
## Make sure `support.values.yaml` is correctly configured
```

When things are very verbose, they can become harder to follow by increasing the amount of text to read and overview. The initial paragraph below clarifies the full path still, so i figure this is a good place to cut down on the verbosity.